roles/nextflow/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 java_home: /sw/comp/java/x86_64/OracleJDK_11.0.9
 nextflow_java: "{{ java_home }}"
 nextflow_version_tag: 23.10.0
-nextflow_download_url: https://github.com/nextflow-io/nextflow/releases/download/v{{ nextflow_version_tag }}/nextflow-{{ nextflow_version_tag }}-all
+nextflow_download_url: https://github.com/nextflow-io/nextflow/releases/download/v{{ nextflow_version_tag }}/nextflow
```

Comment:
are you sure about this one? I know we had problems with running the pipelines offline previously without the `-all` version

Reply:
That's what we are using with tower now. Matthias said the `all` binaries have been deprecated by Seqera.